src/cryptonote_core/blockchain.cpp

```diff
@@ -147,6 +147,7 @@ static const struct {
   { 9, 1057778, 0, 1533297600 },
   { 10, 1154318, 0, 1550153694 },
   { 11, 1155038, 0, 1550225678 },
+  { 12, 1290420, 0, 1567378800 },	/* FIXME: set actual height and datestamp */
```

Comment:
Was the actual testnet height chosen already or it's a temporary value?

Reply:
I considered this a temporary value. That's why the FIXME comment is still there.